container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/kubernetes_label_key/label_key_consts.go

```diff
@@ -33,11 +33,11 @@ const (
 	// the logs aggregator (vector) needs the enclave uuid and service uuid label keys to create the filepath where logs are stored in persistent volume
 	// but vectors template syntax can't interpret the "kurtosistech.com/" prefix, so we can't use the existing label keys or their prefix
 	// to avoid collisions with labels the user may add, kurtosis_ prefix is added
-	logsOnlyKurtosisPrefix                     = "kurtosis_"
-	logsOnlyEnclaveUuidLabelKeyStr             = logsOnlyKurtosisPrefix + "enclave_uuid"
-	logsOnlyServiceUuidKubernetesLabelKey      = logsOnlyKurtosisPrefix + "service_uuid"
-	logsOnlyServiceShortUuidKubernetesLabelKey = logsOnlyKurtosisPrefix + "service_short_uuid"
-	logsOnlyServiceNameKubernetesLabelKey      = logsOnlyKurtosisPrefix + "service_name"
+	logsOnlyKurtosisPrefix                = "kurtosis_"
+	logsOnlyEnclaveUuidLabelKeyStr        = logsOnlyKurtosisPrefix + "enclave_uuid"
+	logsOnlyServiceUuidKubernetesLabelKey = logsOnlyKurtosisPrefix + "service_uuid"
+
+	engineNodeLabelKeyStr = labelKeyPrefixStr + "engine-node"
```

Comment:
Seems like we (KT) are labelling the node. Shouldn't that be the user's responsibility, and we just use whatever node they tell us?

Reply:
Hm I'd prefer if Kurtosis managed that label for the user actually, one less step for a user to do when they switch to using the k8s backend. As long as Kurtosis is only interacting with labels it owns, we wouldn't interfere with other labels on a users node.